App/isowords.xcodeproj/project.pbxproj

```diff
@@ -1339,7 +1339,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
```

Comment:
To use `navigationDestination` I've bumped the deployment target. Up for alternate ideas, though.

Reply:
nah iOS 16 sounds good to me!